### ISSUE TITLE
[Bugs]  lr_mult, decay_mult unavailable in optimizer

### ISSUE THREAD

**Issue by czqInNanjing** [mit-han-lab/temporal-shift-module#108]
Hi, I find a bug in the code: ops/models.py
In pytorch, lr_mult, decay_mult is not available parameter, I guess this is because the original code is transformed from caffe which support these paramters.

```python
        return [
            {'params': first_conv_weight, 'lr_mult': 5 if self.modality == 'Flow' else 1, 'decay_mult': 1,
             'name': "first_conv_weight"},
            {'params': first_conv_bias, 'lr_mult': 10 if self.modality == 'Flow' else 2, 'decay_mult': 0,
             'name': "first_conv_bias"},
            {'params': normal_weight, 'lr_mult': 1, 'decay_mult': 1,
             'name': "normal_weight"},
            {'params': normal_bias, 'lr_mult': 2, 'decay_mult': 0,
             'name': "normal_bias"},
            {'params': bn, 'lr_mult': 1, 'decay_mult': 0,
             'name': "BN scale/shift"},
            {'params': custom_ops, 'lr_mult': 1, 'decay_mult': 1,
             'name': "custom_ops"},
            # for fc
            {'params': lr5_weight, 'lr_mult': 5, 'decay_mult': 1,
             'name': "lr5_weight"},
            {'params': lr10_bias, 'lr_mult': 10, 'decay_mult': 0,
             'name': "lr10_bias"},
        ]
```

**Comment by tonylins** [comment]
Hi, I agree that the code piece from TSN is a little hack to support different learning rate, but it should work fine with PyTorch. Do you find that it leads to bugs when running the code?

**Comment by tonylins** [comment]
The related code is here: https://github.com/mit-han-lab/temporal-shift-module/blob/master/main.py#L362

**Comment by czqInNanjing** [comment]
Yes, the code can run. 
However, the code mislead people who want to adjust the lr of different part of the layers. I am doing transfer learning projects and need to set the backbone  with smaller lr. I changed the parameter of lr_multi but it does not work and it costs me lots of time to fix the bug.

Since this project  already have near 1k stars, I guess it would be better to fix the bug so that it would not mislead other developer.

Again, thanks for your great work.

**Comment by tonylins** [comment]
Yes, I will refactor this part to make it more clear when time permits.
